README.md

```diff
@@ -2,69 +2,147 @@
 
 # React Native Calendars 🗓️ 📆
 
+## A declarative cross-platform React Native calendar component for iOS and Android.
+
 [![Version](https://img.shields.io/npm/v/react-native-calendars.svg)](https://www.npmjs.com/package/react-native-calendars)
 [![Build Status](https://travis-ci.org/wix/react-native-calendars.svg?branch=master)](https://travis-ci.org/wix/react-native-calendars)
 
-This module includes various customizable **React-Native** calendar components.
+<br>
+
+This module includes information on how to use this customizable **React-Native** calendar component.
+
+The package is compatible with both **Android** and **iOS**
+
+<br>
+
+
+> ### **Official documentation**  
+> This README provides basic examples of how to get started with `react-native-calendars`. For detailed information, refer to the [official documentation site](https://wix.github.io/react-native-calendars/docs/intro).
 
-The package is both **Android** and **iOS** compatible.
 
-**See our new [Docs site](https://wix.github.io/react-native-calendars/)**
+## Features ✨
+* Pure JS. No Native code required
```

Comment:
We should add the marking dates (dot, multi-dot, period, multi-period and custom) options

Reply:
Good point! Will add them.